cluster/juju/layers/kubernetes-master/actions.yaml

```diff
@@ -0,0 +1,19 @@
+create-rbd-pv:
```

Comment:
it was suggested to parameterize the pool, its not likely to need tweaking but in the event a storage admin has provisioned a specific pool for kubernetes, we should at least have an answer for how to tune the command